diff --git a/src/index.ts b/src/index.ts
index 33d5af3..bd5746d 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -941,8 +941,8 @@ import { PlayerTeleportWaystoneStart } from "./player_teleport_waystone_start_re
 export { PlayerTeleportWaystoneStart };
 import { PlayerUseElevator } from "./player_use_elevator_reducer.ts";
 export { PlayerUseElevator };
-import { PlayerVoteAnswer } from "./player_vote_answer_reducer.ts";
-export { PlayerVoteAnswer };
+import { PlayerVoteAnswer as PlayerVoteAnswerReducer } from "./player_vote_answer_reducer.ts";
+export { PlayerVoteAnswerReducer };
 import { PlayerVoteConclude } from "./player_vote_conclude_reducer.ts";
 export { PlayerVoteConclude };
 import { PocketSwapContents } from "./pocket_swap_contents_reducer.ts";
